src/features/eta/FavouriteEta.tsx

```diff
@@ -49,7 +63,11 @@ export default function FavouriteEta() {
         return;
       }
       setData(parsedData);
-      setEtaDb(multiStopParser(parsedData));
+      if (unifiedEtaValue) {
+        setUnifiedEtaDb(multiStopUnifier(parsedData, stopBookmarks));
+      } else {
+        setSingleEtaDb(multiStopParser(parsedData));
+      }
```

Comment:
Can we do it in one liner?

```
unifiedEtaValue ? setUnifiedEtaDb(multiStopUnifier(parsedData, stopBookmarks)) : setSingleEtaDb(multiStopParser(parsedData))
```

Reply:
I want to keep inline ifelse to a minimum for clarity (usually only used within rendering)

Reply:
one-liner ifelse should be applied when there is only one line in each if/else scope